### PR TITLE
Elastio fargate stream scheduled backup

### PR DESCRIPTION
**Pull request by ViacheslavSaievskyi**

### CODE REVIEW

**Review by anelson — CHANGES_REQUESTED** [review]

**Comment by anelson** [comment]
@ViacheslavSaievskyi what's the status of this PR?  Since I requested changes nothing more seems to have been done.

**Comment by ViacheslavSaievskyi** [comment]
> @ViacheslavSaievskyi what's the status of this PR? Since I requested changes nothing more seems to have been done.

**Review by anelson — CHANGES_REQUESTED** [review]
I made three comments earlier; none of them have been addressed.  Address them and re-request.

Second, all of this content should be in a single sub-folder in `contrib`.  Right now you're adding a bunch of folders to the root which will be confusing and misleading.

Move everything to a folder called `fargate-stream-backup`.

Finally, why are there `mp4` files in this repo?  You should never put large binary files in Git.  We have Git LFS for this purpose if you need them as part of the content.

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,11 @@
 .history/
 
 # Built Visual Studio Code Extensions
-*.vsix
```
Comment (anelson) [inline .gitignore:12]: Don't put this here.  You can have your own `.gitignore` inside of the `elastio-fargate-stream-backup` directory where project-specific ignores go.

**Review by anelson — APPROVED** [review]
Approved, but make sure to move your `.gitignore` as instructed, before you merge.

Also when you merge, edit the commit message so it's meaningful.  Right now it will default to a mess of short commit messages reflecting all of the changes you made up to this point.  We don't want that, we want it squashed into a clean description of this change.

**Pull request merged**